custom-graders/DemoAssignmentGrader/GraderFiles/executeGrader.sh

```diff
@@ -0,0 +1,75 @@
+#! /bin/bash
+
+# Coursera deletes all environment variables set inside 'Dockerfile'. If any environment varables
+# need to be set, they must be set inside a wrapper bash script.
+export JAVA_HOME=/usr/lib/jvm/java-7-openjdk-amd64
+
+# Switch to the grader directory
+cd /grader
+
+# Compile the learner's program in the current directory. We can safely assume that there
+# would be a single submission file in this directory.
+javac -d . /shared/submission/*
+
+# Note: Nothing except Json object containing 'fractionalScore' and 'feedback' should be written
+# to stdout.
+
+# Check if the compilation was successful
+if [ ! $? -eq 0 ]; then
+    echo "{ \"fractionalScore\":0.0, \"feedback\":\"Compile Error\" }"
+    exit 0
+fi
+
+# Parse the command line arguments supplied by Coursera.
```

Comment:
Why do you parse command line parameters after compiling? Please parse CLI params first. Thanks!